src/hist/basehist.py

```diff
@@ -401,20 +403,45 @@ def plot2d_full(
 
         return hist.plot.plot2d_full(self, ax_dict=ax_dict, **kwargs)
 
+    def plot_ratio(
+        self,
+        other: Callable[[np.ndarray], np.ndarray],
+        *,
+        ax_dict: "Optional[Dict[str, matplotlib.axes.Axes]]" = None,
+        **kwargs: Any,
+    ) -> "Tuple[MainAxisArtists, RatiolikeArtists]":
+        """
+        ``plot_ratio`` method for ``BaseHist`` object.
+
+        Return a tuple of artists following a structure of
+        ``(main_ax_artists, subplot_ax_artists)``
+        """
+
+        import hist.plot
+
+        return hist.plot._plot_ratiolike(
+            self, other, ax_dict=ax_dict, view="ratio", **kwargs
+        )
+
     def plot_pull(
         self,
         func: Callable[[np.ndarray], np.ndarray],
```

Comment:
```suggestion
        func: Union[str, Callable[[np.ndarray], np.ndarray]],
```

?

Reply:
Ah yeah, I guess that got missed. Strange that it didn't complain. :?

Reply:
Addition of `str` is causing `mypy` to explode everywhere, so this is going to take some debugging. :/ I'll request another review once done, but this won't be fast.